translations/wesnoth-A_New_Order/es.po

```diff
@@ -177,16 +193,22 @@ msgid ""
 "fights most of the time. They also usually build or buy (or steal) a better "
 "crossbow, and their accuracy and skill have both improved with practice."
 msgstr ""
+"Después de cuatro o cinco robos, los bandoleros que duran tanto suelen "
+"encontrar una espada de algún tipo. Normalmente se trata de un sable, porque "
+"los esgrimistas son sus presas más habituales, ya que suelen vagar por "
+"caminos solitarios en busca de pelea la mayor parte del tiempo. También "
+"suelen construir o comprar (o robar) una ballesta mejor, y tanto su "
+"precisión como su habilidad han mejorado con la práctica."
 
 #. [attack]: type=blade
 #: A_New_Order/units/others/Outlaw_Raider.cfg:38
 msgid "saber"
-msgstr ""
+msgstr "sable"
 
 #. [unit_type]: id=Brigand, race=human
 #: A_New_Order/units/others/Brigand.cfg:5
 msgid "Brigand"
-msgstr ""
+msgstr "Bandolero"
```

Comment:
This term has kind of made its way into American Spanglish as a term associated with cowboys; I usually picture them wearing one of these: https://en.wikipedia.org/wiki/Bandolier
...does that association exist in actual Spanish, too?

Reply:
That's a "bandolera" in spanish, basically the same term in feminine xD

Reply:
ok, just wondering